app/scripts/proactive/view/CatalogPublishView.js

```diff
@@ -202,8 +203,12 @@ define(
         },
         render: function () {
             this.$el.html(this.template());
-            this.updateBuckets();
-            this.buckets.on('sync', this.updateBuckets, this);
+            var bucketKind = this.kind;
+            //for workflows, we don't want subkind filters
+            if (this.kind.toLowerCase().indexOf('workflow') > -1)
```

Comment:
same comment as above

Reply:
not sure this is robust as you search for workflow anywhere in the string.
If the workflow kind are:
workflow/standard and workflow/pca

To avoid conflicts with user-defined kinds, it would probably be better to use:
if (this.kind.toLowerCase().indexOf('workflow') == 0) {
}
We now the workflow kinds starts with workflow